src/seedu/addressbook/commands/ClearCommand.java

```diff
@@ -16,4 +16,6 @@ public CommandResult execute() {
         addressBook.clear();
         return new CommandResult(MESSAGE_SUCCESS);
     }
+    public boolean isMutating() { return true;}
```

Comment:
Better to put `@Override` annotation so that it's clearer for other programmers.